airflow/utils/db.py

```diff
@@ -1074,9 +1073,10 @@ def _validate_version_range(command, config, version_range):
         raise AirflowException(
             'MSSQL is not supported for offline migration in Airflow versions less than 2.2.0.'
         )
-    revision = f"{REVISION_HEADS_MAP[lower]}:{REVISION_HEADS_MAP[upper]}"
+    _lower, _upper = REVISION_HEADS_MAP[lower], REVISION_HEADS_MAP[upper]
+    revision = f"{_lower}:{_upper}"
     try:
-        command.history(config, rev_range=revision)
+        list(script_.revision_map.iterate_revisions(_upper, _lower))
```

Comment:
What does this change?

Reply:
Currently, if you run the `airflow db upgrade --range 2.0.0:2.2.3` it prints migration history before printing the SQL. Something like this:
```
64a7d6477aae -> e959f08ac86c, Change field in ``DagCode`` to ``MEDIUMTEXT`` for MySql
30867afad44a -> 54bebd308c5f, Adds ``trigger`` table and deferrable operator columns to task instance
e9304a3141f0 -> 30867afad44a, Rename ``concurrency`` column in ``dag`` table to`` max_active_tasks``
83f031fd9f1c -> e9304a3141f0, Make XCom primary key columns non-nullable
ccde3e26fe78 -> 83f031fd9f1c, Improve MSSQL compatibility
092435bf5d12 -> ccde3e26fe78, Add index on state, dag_id for queued ``dagrun``
```
So the above code does this internally and does not print the history.

Reply:
I see. Perhaps add a comment to describe what this line does then? (Run through the revisions to check they all execute correctly?)